tests/perf-test/ReportScreen.perf-test.tsx

```diff
@@ -91,13 +105,12 @@ beforeAll(() =>
     Onyx.init({
         keys: ONYXKEYS,
         safeEvictionKeys: [ONYXKEYS.COLLECTION.REPORT_ACTIONS],
-        registerStorageEventListener: () => {},
     }),
 );
 
 // Initialize the network key for OfflineWithFeedback
 beforeEach(() => {
-    global.fetch = TestHelper.getGlobalFetchMock();
+    global.fetch = TestHelper.getGlobalFetchMock() as typeof fetch;
```

Comment:
```suggestion
    // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
    global.fetch = TestHelper.getGlobalFetchMock();
```

It would be better to just suppress the error and comment that we need to remove the comment once `TestUtils` is migrated